packages/ui-spinner/src/Spinner/__tests__/Spinner.test.js

```diff
@@ -48,10 +48,16 @@ describe('<Spinner />', async () => {
   })
 
   it('should render the contents of a component used in renderTitle', async () => {
-    const Translation = ({children}) => <span>I have translated {children}.</span>
-    await mount(<Spinner renderTitle={<Translation>Loading</Translation>} size="small" />)
+    const Translation = ({ children }) => (
+      <span>I have translated {children}.</span>
+    )
+    await mount(
+      <Spinner renderTitle={<Translation>Loading</Translation>} size="small" />
+    )
     const spinner = await SpinnerLocator.find()
-    expect(await SpinnerLocator.find(':contains(I have translated Loading)')).to.exist()
+    expect(
+      await SpinnerLocator.find(':contains(I have translated Loading)')
+    ).to.exist()
```

Comment:
We should use common linting rules that are auto-applied at latest when committing code to prevent formatting-only changes.

Reply:
This change was made by prettier, we will continue to use auto linting on commit (and optionally in the IDE too).